src/lib/http.coffee

```diff
@@ -21,12 +23,94 @@ module.exports = (dnschain) ->
             @log = gNewLogger 'HTTP'
             @log.debug "Loading HTTPServer..."
             @rateLimiting = gConf.get 'rateLimiting:http'
+            app = express()
+            ld = _ # stored for use after overwriting _
```

Comment:
maybe try just using two underscores instead of one for ignoring vars during destructuring? that's what i've done in the past to preserve lodash's underscore.